### ISSUE TITLE
Improve minProcessorVersion

### ISSUE THREAD

**Issue by AndreasGassmann** [Acurast/acurast-cli#51]
We could
- print this list by an info command or flag on commander
- print this list if a invalid SemVer is provided

_Originally posted by @gitsimon in https://github.com/Acurast/acurast-cli/pull/50#discussion_r1808478242_

**Comment by AndreasGassmann** [comment]
I agree. I also think we should dynamically load this list in the future (eg. to make developers aware if there is a new version and they could increase the min version). I'll create a new issue for this.

**Comment by kaltra-muho** [comment]
Implemented this issue with benchmark filter
